frontend/packages/ceph-storage-plugin/src/components/kms-config/kms-config.tsx

```diff
@@ -63,7 +63,8 @@ export const KMSConfigure: React.FC<KMSConfigureProps> = ({ state, dispatch, mod
 
   const setAddressPort = (port: string) => {
     kmsObj.port.value = port;
-    kmsObj.port.valid = port !== '' && !_.isNaN(Number(port)) && Number(port) > 0;
+    kmsObj.port.valid =
```

Comment:
Is patternfly is providing any port component?

Reply:
I checked patternfly only supports following types for `<TextInput>`:
```
"number" | "time" | "text" | "url" | "date" | "datetime-local" | "email" | "month" | "password" | "search" | "tel"
```